corehq/apps/app_manager/views/phone.py

```diff
@@ -0,0 +1,70 @@
+from django.http import HttpResponse
+from eulxml.xmlmap import XmlObject
+
+from django.conf import settings
+
+from corehq.apps.domain.auth import basicauth
+from corehq.apps.domain.decorators import check_lockout
+from corehq.apps.users.models import CouchUser
+from corehq.apps.app_manager.dbaccessors import get_built_app_ids, get_app
+from corehq.apps.app_manager.suite_xml.xml_models import (
+    StringField,
+    IntegerField,
+    NodeListField,
+)
+
+
+class App(XmlObject):
+    ROOT_NAME = 'app'
+
+    domain = StringField('@domain')
+    name = StringField('@name')
+    environment = StringField('@environment')
+    version = IntegerField('@version')
+    media_profile = StringField('@media-profile')
+    profile = StringField('@profile')
+
+
+class AppList(XmlObject):
+    ROOT_NAME = 'apps'
+    apps = NodeListField('app', App)
+
+
+@check_lockout
+@basicauth()
```

Comment:
what's the benefit of restricting this to basic auth? as opposed to using `login_or_digest_or_basic_or_apikey` or similar